src/domain/session/room/timeline/tiles/GapTile.js

```diff
@@ -55,7 +70,18 @@ export class GapTile extends SimpleTile {
         let canFillMore;
         this._siblingChanged = false;
         do {
-            canFillMore = await this.fill();
+            try {
+                canFillMore = await this.fill();
+            }
+            catch (e) {
+                if (e instanceof ConnectionError) {
```

Comment:
how could we ever have a connection error here if we catch that and drop the exception in `fill` already?

Reply:
@MidhunSureshR Could you add a word why your resolved this?

Reply:
We rethrow the same exception after handling it?

Reply:
Right, for some reason I thought the `throw err` was in an else case :facepalm: